enterprise/coordinator/recovery/recovery.go

```diff
@@ -0,0 +1,180 @@
+// Copyright 2025 Edgeless Systems GmbH
+// SPDX-License-Identifier: BUSL-1.1
+
+package recovery
```

Comment:
should we call this autorecovery to distinguish it from the cli-based recovery?

Reply:
Renamed it to `peerrecovery` to avoid confusion with a potential recovery from sealed secrets.